packages/common/workspace/src/engine/blob.ts

```diff
@@ -12,6 +12,7 @@ const logger = new DebugLogger('affine:blob-engine');
  */
 export class BlobEngine {
   private abort: AbortController | null = null;
+  private isOverCapacity: boolean = false;
```

Comment:
You need an event to tell the outside about this change.

example: https://github.com/toeverything/AFFiNE/blob/93a2398101e3300828822d87144b0c559b4f4885/packages/common/workspace/src/engine/sync/engine.ts#L58https://github.com/toeverything/AFFiNE/blob/93a2398101e3300828822d87144b0c559b4f4885/packages/common/workspace/src/engine/sync/engine.ts#L58

Reply:
Do the latest changes meet your expectations?

Reply:
exactly